manual_correction.py

```diff
@@ -87,31 +87,27 @@ def get_parser():
             """)
     )
     parser.add_argument(
-        '-path-in',
+        '-path-img',
         metavar="<folder>",
         required=True,
-        help='Path to the processed data. Example: ~/<your_dataset>/data_processed or ~/<your_dataset>/data_to_correct',
+        help='Path to the folder where niftii images are stored. Example: ~/<your_dataset>/data_processed',
```

Comment:
I would keep the original description:

```suggestion
        help'Path to the folder with processed data (BIDS-compliant). Example: ~/<your_dataset>/data_processed or ~/<your_dataset>/data_to_correct',
```